package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "smartcar-unit",
-  "version": "1.1.3",
+  "version": "0.0.0-semantic-release",
```

Comment:
what is this doing?

Reply:
the `package.json` file in the git repo never has the correct version, `semantic-release` determines what the version needs to be when it publishes to npm. So whenever you install from npm the correct version is there, but we shouldn't put it here so there is no confusion in case it's out of sync with what is published.